Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -127,7 +127,7 @@
   name: retro laser blaster
   parent: [ BaseWeaponBatterySmall, BaseMajorContraband ]
   id: WeaponLaserGun
-  description: A weapon using light amplified by the stimulated emission of radiation.
+  description: A weapon using light amplified by the stimulated emission of radiation. Defeats glass cover, passing right through.
```

Comment:
also the wording in this sentence is really clunky if you do opt to keep it. smth like "Passes right through glass." or "Shoots through glass windows." would be more clear

Reply:
Yeah, clarity would make sense; I chose "defeats cover" because that's the more, utility description. Clarity above all else.
And I was thinking of that fact, with players finding out ballistic vs laser vs bolts. Will consider removing it